src/Components/Search/NewSearch/NewSearchBarInput.tsx

```diff
@@ -161,16 +201,18 @@ export const NewSearchBarInputQueryRenderer: FC<NewSearchBarInputQueryRendererPr
         query NewSearchBarInputSuggestQuery(
           $term: String!
           $hasTerm: Boolean!
+          $entities: [SearchEntity]
         ) {
           viewer {
             ...NewSearchBarInput_viewer
-              @arguments(term: $term, hasTerm: $hasTerm)
+              @arguments(term: $term, hasTerm: $hasTerm, entities: $entities)
           }
         }
       `}
       variables={{
         hasTerm: false,
         term: "",
+        entities: null,
```

Comment:
**suggestion:** always to pass entities (feel free to ignore suggestions). Something like this

```javascript
const DEFAULT_PILL = TOP_PILL
const DEFAULT_PILL_ENTITY = ELASTIC_PILL_KEY_TO_SEARCH_ENTITY[DEFAULT_PILL.key]

...

variables={{
  hasTerm: false,
  term: "",
  entities: [DEFAULT_PILL_ENTITY],
}}
```

Reply:
There is no TOP_PILL search entity, that's why I pass null when TOP pill is selected: https://github.com/artsy/metaphysics/blob/17e599f15acc745d39a3b6cf664b5dedbc66c78a/_schemaV2.graphql#L14667

Reply:
**suggestion**: I think in this case it is better to pass an empty array instead of `null` (as it is done [here](https://github.com/artsy/force/blob/c578504f29a91dc1d6da1c54fa2ecf86672b7cb8/src/Components/Search/NewSearch/NewSearchBarInput.tsx#L106)) so that there is the same logic everywhere